### DISCUSSION TITLE
ScottPlot

### DISCUSSION THREAD

**Discussion by kookma** [DanWBR/dwsim#409] [Ideas]
I found this plot library interesting!

https://github.com/ScottPlot/ScottPlot

It has MIT License. It may be of potential use in DWSIM.

**Comment by DanWBR** [comment]
@kookma DWSIM uses ZedGraph (https://github.com/ZedGraph/ZedGraph) for Classic UI and OxyPlot (https://github.com/oxyplot/oxyplot) for Cross-Platform UI.
